features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NRF5/source/nRF5xGattServer.h

```diff
@@ -79,19 +102,52 @@ class nRF5xGattServer : public GattServer
         return -1;
     }
 
+    /**
+     * Return the biggest size used by a characteristic in the server
+     */
+    uint16_t getBiggestCharacteristicSize() const;
+
+    /**
+     * Allocate a new write long request. return null if no requests are available.
+     * @param  connection_handle The connection handle which where the request will
```

Comment:
"which where the request will happen" is hardly understandable. Maybe something like "to be associated with the request"?